index.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 var got = require('got');
-var registryUrl = require('registry-url');
+var npmKeywordUrl = require('npm-keyword-url');
```

Comment:
I'm all for modularity, but this feels like a bit too much. I see no use for this for anything else.